### DISCUSSION TITLE
DHCP/DNS 禁用解析ipv6后，解析ipv4也被关掉了。

### DISCUSSION THREAD

**Discussion by sady2009-wq** [Boos4721/openwrt#20] [Bugs/问题反馈]
发现一个bug,DHCP/DNS 禁用解析ipv6后，解析ipv4也被关掉了。

**Comment by Boos4721** [comment]
**FULL LOG**

**Comment by sady2009-wq** [comment]
Sat Sep 25 08:29:12 2021 kern.info kernel: [41814.264299] nss-dp 3a001600.dp4 eth2: PHY Link up speed: 1000
Sat Sep 25 08:29:12 2021 kern.info kernel: [41814.264386] br-lan: port 2(eth2) entered blocking state
Sat Sep 25 08:29:12 2021 kern.info kernel: [41814.269131] br-lan: port 2(eth2) entered forwarding state
Sat Sep 25 08:29:12 2021 daemon.notice netifd: Network device 'eth2' link is up
Sat Sep 25 08:29:15 2021 kern.info kernel: [41817.332131] nss-dp 3a001600.dp4 eth2: PHY Link is down
Sat Sep 25 08:29:15 2021 kern.info kernel: [41817.332676] br-lan: port 2(eth2) entered disabled state
Sat Sep 25 08:29:15 2021 daemon.notice netifd: Network device 'eth2' link is down
Sat Sep 25 08:29:21 2021 kern.info kernel: [41823.476193] nss-dp 3a001600.dp4 eth2: PHY Link up speed: 1000
Sat Sep 25 08:29:21 2021 kern.info kernel: [41823.476280] br-lan: port 2(eth2) entered blocking state
Sat Sep 25 08:29:21 2021 kern.info kernel: [41823.480944] br-lan: port 2(eth2) entered forwarding state
Sat Sep 25 08:29:21 2021 daemon.notice netifd: Network device 'eth2' link is up
Sat Sep 25 08:48:23 2021 daemon.crit dnsmasq[10462]: bad option at line 14 of /var/etc/dnsmasq.conf.cfg01411c
Sat Sep 25 08:48:23 2021 daemon.crit dnsmasq[10462]: FAILED to start up
Sat Sep 25 08:48:23 2021 daemon.info procd: Instance dnsmasq::cfg01411c s in a crash loop 6 crashes, 0 seconds since last crash

**Comment by sady2009-wq** [comment]
![1](https://user-images.githubusercontent.com/69339069/134752926-9745e285-2722-4ef9-984a-4c2d071c7449.png)
![2](https://user-images.githubusercontent.com/69339069/134752929-ae8ece82-49fa-43e5-8840-b0652a181091.png)

**Reply by Boos4721** [reply]
**FIX LATER**

**Comment by Boos4721** [comment]
**FIXED**
